pandas/tseries/offsets.py

```diff
@@ -1040,54 +1040,52 @@ def __init__(self, n=1, normalize=False, weekmask='Mon Tue Wed Thu Fri',
 
     @cache_readonly
     def cbday(self):
-        kwds = self.kwds
-        return CustomBusinessDay(n=self.n, normalize=self.normalize, **kwds)
+        cbday = CustomBusinessDay(n=self.n, normalize=False, **self.kwds)
+
+        # Define default roll function to be called in apply method
```

Comment:
why not simply define ``roll_func`` (better name?) as a property (or function) or CBD?